docs/delorean/early-warning-system.md

```diff
@@ -0,0 +1,72 @@
+## Delorean Early Warning System
+
+## Table of Contents
+  - [1. Overview](#Overview)
+  - [2. Job Structure](#Job Structure)
```

Comment:
This isn't rendering correctly. Spaces needs to be replaced by `-`. `(#job-structure)`. Same with `Job Types`